src/Tickets/Commerce/Gateways/Stripe/Settings.php

```diff
@@ -147,21 +147,47 @@ private function set_connection_status() {
 	 * @inheritDoc
 	 */
 	public function get_settings() {
+		$currency_code = Currency::get_currency_code();
+		$currency_name = Currency::get_currency_name( $currency_code );
+		$plus_link     = sprintf(
+			'<a href="https://evnt.is/19zl" target="_blank" rel="noopener noreferrer">%s</a>',
+			esc_html__( 'Learn more', 'event-tickets' )
+		);
```

Comment:
Does this need translator's notes, too?

Reply:
No, because we're not adding placeholders to the string that is translated. We should add translator notes when the string that is translated has some placeholder.

The point of the translator's notes is normally to provide more information to translators of what's a placeholder in a particular string. So that when they see the string on poedit or glotpress or any other place, they have more context on what's inside that weird `%1d` or something.